### PR TITLE
chore(tslint): disable select tslint rules

### PR DESCRIPTION
**Pull request by davemneo**
- ban-ts-ignore
 - no-default-import

### Description of the Change
Update the tslint.json to disable selected rules. 

### Applicable Issues
https://github.com/neo-one-suite/neo-one/pull/852

### CODE REVIEW

**Comment by davemneo** [comment]
bors cancel

**Comment by davemneo** [comment]
bors try

File: tslint.json [hunk 1]
```diff
@@ -168,6 +168,9 @@
     "use-primitive-type": true,
     "no-map-without-usage": true,
     "rxjs-finnish": true,
-    "rxjs-no-ignored-error": false
+    "rxjs-no-ignored-error": false,
+    "ban-ts-ignore": false,
+    "no-default-import": false,
+    "callable-types": false
```
Comment (dicarlo2) [inline tslint.json]: I think this one we can leave on

Comment (davemneo) [reply tslint.json]: Will remove. 

I added it for this case:

image

**Pull request merged**